src/ibm/hypervisor-network-mgr-src/hyp_network_manager.cpp

```diff
@@ -244,17 +249,17 @@ void HypNetworkMgr::createIfObjects()
     // created during init time to support the static
     // network configurations on the both.
     // create eth0 and eth1 objects
-    log<level::INFO>("Creating eth0 and eth1 objects");
+    log<level::ERR>("Creating eth0 object");
     interfaces.emplace("eth0",
                        std::make_unique<HypEthInterface>(
                            bus, (objectPath + "/eth0").c_str(), "eth0", *this));
+    interfaces["eth0"]->createIPAddressObjects();
+    if(if1Present){
+    log<level::ERR>("Creating eth1 object");
```

Comment:
Why createIPAddressObjects is not called for eth1 ?

Reply:
added